apps/charterafrica/src/payload/blocks/VideoEmbed.js

```diff
@@ -0,0 +1,81 @@
+import blockFields from "../fields/blockFields";
+
+const VideoEmbed = {
+  slug: "videoEmbed",
+  fields: [
+    blockFields({
+      name: "videoEmbedField",
+      fields: [
+        {
+          type: "row",
+          fields: [
+            {
+              name: "embedType",
+              type: "radio",
+              defaultValue: "url",
+              options: [
+                {
+                  label: {
+                    en: "URL",
+                    fr: "URL",
+                    pt: "URL",
+                  },
+                  value: "url",
+                },
+                {
+                  label: {
+                    en: "Code",
+                    fr: "Code",
+                    pt: "Código",
+                  },
+                  value: "code",
+                },
+              ],
+            },
+          ],
+        },
+        {
+          name: "url",
+          label: {
+            en: "Video URL",
+            fr: "URL de la vidéo",
+            pt: "URL do vídeo",
+          },
```

Comment:
Thinking out loud, would embedding video, audio or even a chart be any different on the front-end?

Reply:
I think they'd follow a similar approach @kilemensi